### PR TITLE
fix devtools stage

### PR DESCRIPTION
**Pull request by hyp**
Fix the build failures in the devtools stage

### CODE REVIEW

File: .github/workflows/swift-toolchain.yml [hunk 1]
```diff
@@ -2416,7 +2410,7 @@ jobs:
                 -D CMAKE_Swift_COMPILER=${SWIFTC} `
                 -D CMAKE_Swift_COMPILER_TARGET=${{ matrix.triple }} `
                 -D CMAKE_Swift_COMPILER_WORKS=YES `
-                -D CMAKE_Swift_FLAGS="${{ inputs.CMAKE_Swift_FLAGS }}" `
```
Comment (Steelskin) [inline .github/workflows/swift-toolchain.yml:2419]: Should this be appended rather than replaced?
(Same for all other instances of this replacement)

Comment (hyp) [reply .github/workflows/swift-toolchain.yml:2419]: appended where?

**Review by compnerd — APPROVED** [review]
Ugh, is really unfortunate that just picking it up from the environment doesn't work equivalently.

File: .github/workflows/swift-toolchain.yml (hunk 1 above)
Comment (Steelskin) [reply .github/workflows/swift-toolchain.yml:2419]: Oh nevermind, I was on mobile and misread the diff.

**Review by Steelskin — APPROVED** [review]

**Review by hjyamauchi — APPROVED** [review]

**Review by Steelskin — APPROVED** [review]

**Pull request merged**